.github/workflows/ci.yml

```diff
@@ -88,7 +88,7 @@ jobs:
           - target: x86_64-apple-darwin
             os: macos-latest
           - target: x86_64-unknown-linux-gnu
-            os: ubuntu-latest
+            os: ubuntu-20.04
```

Comment:
Can you add a comment above explaining why we are building on 20.04 here? Otherwise people might think it's inconsistent with the other ubuntu versions in the file and want to unify it.